packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -200,6 +201,11 @@ class CupertinoDialogAction extends StatelessWidget {
   /// If this is set to null, the button will be disabled.
   final VoidCallback onPressed;
 
+  /// Set to true if button is the default choice in the dialog.
+  ///
+  /// Default buttons are bolded.
```

Comment:
Apple seems to use the same wording https://developer.apple.com/ios/human-interface-guidelines/ui-views/alerts/

Renamed to isDefaultAction to be more specific. Renamed isDestructive to isDestructiveAction too for categorisation